platform/pulpcore/app/models/publication.py

```diff
@@ -0,0 +1,103 @@
+from django.db import models
+from django.core.files.storage import DefaultStorage
+
+from pulpcore.app.models import Model
+
+
+class Publication(Model):
+    """
+    Fields:
+        created (models.DatetimeField): When the publication was created UTC.
+
+    Relations:
+        publisher (models.ForeignKey): The publisher that created the publication.
+    """
+
+    created = models.DateTimeField(blank=True, null=True, auto_now_add=True)
+
+    publisher = models.ForeignKey('Publisher', on_delete=models.CASCADE)
+
+
+class PublishedFile(Model):
+    """
+    A file included in Publication.
+
+    Fields:
+        relative_path (models.CharField): The (relative) path component of the published url.
+
+    Relations:
+        publication (models.ForeignKey): The publication in which the artifact is included.
+
+    """
+    relative_path = models.CharField(blank=False, null=False, max_length=255)
```

Comment:
Is this a required field? I expected every publication is required to have a `relative_path` set.